packages/react-hooks/src/useSetInterval.test.tsx

```diff
@@ -1,86 +1,63 @@
-import * as React from 'react';
-import { useSetInterval } from './useSetInterval';
-import { safeMount } from '@fluentui/test-utilities';
+import { renderHook } from '@testing-library/react-hooks';
+
 import { validateHookValueNotChanged } from './testUtilities';
+import { useSetInterval } from './useSetInterval';
 
 describe('useSetInterval', () => {
-  // Initialization
-  let timesCalled = 0;
-
-  beforeAll(() => {
+  beforeEach(() => {
     jest.useFakeTimers();
   });
 
-  afterAll(() => {
-    jest.useRealTimers();
-  });
-
   afterEach(() => {
-    timesCalled = 0;
+    jest.useRealTimers();
   });
 
-  const TestComponent = React.forwardRef((props: unknown, ref: React.Ref<{ clearInterval: () => void }>) => {
```

Comment:
this setup was exploding, even using `legacy` timers didn't work. I re-done the test to use more robust approach how to test hooks which make the test pass ✅